superset-frontend/src/chart/Chart.jsx

```diff
@@ -87,6 +87,9 @@ const Styles = styled.div`
     opacity: 0.75;
     font-size: ${({ theme }) => theme.typography.sizes.s}px;
   }
+  svg {
+    width: auto !important;
```

Comment:
```suggestion
    width: auto !important;
```
The above suggestion might also remove the need for `!important`... if that's still not high enough specificity of a selector, we could perhaps play other tricks, like the double ampersand trick, e.g. `&& > svg {`

Reply:
Doesn't work without !important